doc/tutorials/acrn_configurator_tool.rst

```diff
@@ -328,7 +335,10 @@ Save and Check for Errors
       :align: center
       :class: drop-shadow
```

Comment:
line 286 ish -- when defining basic parameters, should a bullet be added that indicates something like.  - parameters that are anticipated to be commonly used for typical ACRN use cases